src/Controllers/CSSGeneratorController.php

```diff
@@ -114,6 +119,32 @@ protected function parseFamilyParameter(string $value): array
         return $result;
     }
 
+    /**
+     * Check font-display value. Value must be one of: auto, block, swap, fallback, optional.
+     *
+     * @param string|null $fontDisplay Font-display value or null
+     * @return string|null Valid font-display css value, or null, if $fontDisplay is null or empty.
+     * @throws \InvalidArgumentException If the parameter set, but has not valid value. The message may be sent
+     *          back to the client.
+     */
+    protected function checkFontDisplay(string $fontDisplay): string
+    {
+        if (!isset($fontDisplay) || $fontDisplay === '') {
+            return null;
+        }
+        $validFontDisplayValues = [
+            'auto',
+            'block',
+            'swap',
+            'fallback',
+            'optional'
+        ];
+        if (!array_search($fontDisplay, $validFontDisplayValues)) {
```

Comment:
Incorrect usage of [array_search](https://www.php.net/manual/en/function.array-search) result:

> Warning
This function may return Boolean FALSE, but may also return a non-Boolean value which evaluates to FALSE. Please read the section on Booleans for more information. Use the === operator for testing the return value of this function.

[in-array](https://www.php.net/manual/en/function.in-array) would fit perfectly. Actually we shouldn't throw an error when the value isn't in the list because:

1. Google Fonts doesn't do it
2. There might be new official or unofficial values for the property in the future and it we would support them "out of the box"

So please just check that the value is either string or null.